### DISCUSSION TITLE
트랜잭션 동시성 제어에 나오는 conflict라는 단어는 무엇을 의미할까요?

### DISCUSSION THREAD

**Discussion by Hanjaemo** [Hanjaemo/DB-Study#14] [Q&A]
트랜잭션 동시성 제어 파트에는 conflict serializable, conflict equivalent 등 conflict라는 단어가 자주 등장합니다.
이 conflict라는 단어가 의미하는 바는 무엇일까요?

**Comment by mithzinf** [comment]
Conflict의 의미

사전적 의미 : 충돌

트랜잭션 동시성 제어 에서의 의미 : 서로 다른 트랜잭션이 동일한 데이터 항목에 대해 동시에 접근하려고 할 때 발생하는 상황

EX) 
'읽기 기능'& '쓰기 기능' 
OR 
'쓰기 기능' - '쓰기 기능' 의 충돌
